nss-day-cohort-30/bangazon-site-suave-salmon
# Rn customers order

Fixes:

Changes proposed in this request & reasons behind organizational or architectural decisions:

Steps to test:
pull down the branch
start the application 
click reports 
click the multiply orders

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
user should appear and number of orders

Commit message:
-

Link to feature ticket:
ticket 10
-

@suave-salmon
